### PR TITLE
Add concat tool to galaxy tools

### PR DESCRIPTION
**Pull request by RichardDavidsonTheGreat**

### CODE REVIEW

**Comment by bgruening** [comment]
Can you use https://github.com/bgruening/galaxytools/blob/master/tools/text_processing/text_processing/cat.xml ?

**Comment by Takadonet** [comment]
> Can you use https://github.com/bgruening/galaxytools/blob/master/tools/text_processing/text_processing/cat.xml ?
The wrapper is 'slightly' differences that takes a (list) paired collection and create a list by concat forward and reverse read. 

@bgruening This tool is a legacy that was only hosted in our internal Toolshed and currently is part of half dozen workflows.

File: tools/concat/.shed.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+categories: [Assembly]
+description: concatenate
+name: concat
+owner: nml
+long_description: concat - Concatenates two files into a resulting file
```
Comment (Takadonet) [inline tools/concat/.shed.yml]: Change long_description to:  **concat - Concatenates (list) paired collection into a single list**

File: tools/concat/.shed.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+categories: [Assembly]
+description: concatenate
+name: concat
```
Comment (Takadonet) [inline tools/concat/.shed.yml]: change name to **concat_paired** and change tool directory name as well to match

File: tools/concat/concat_fastqs.xml [hunk 3]
```diff
@@ -0,0 +1,29 @@
+<tool id="concat_fastqs" name="Concat R1 and R2 fastq" version="0.2">
+  <description>Combine forward and reverse fastqs files </description>
+  <command>
+	cat $texts.forward $texts.reverse > $output
```
Comment (Takadonet) [inline tools/concat/concat_fastqs.xml]: single quotes around all variables.

File: tools/concat/concat_fastqs.xml [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  </inputs>
+  <outputs>
+    <data name="output"  format="fastqsanger" label="$texts.name"></data>
+  
+  </outputs>
+  <tests>
+	  <test>
+	    <param name="texts">
+        <collection type="paired">
+          <element name="forward" value="forward.fastq" />
```
Comment (Takadonet) [inline tools/concat/concat_fastqs.xml]: reduce test files size.

File: tools/concat/.shed.yml [hunk 5]
```diff
@@ -0,0 +1,7 @@
+categories: [Assembly]
+description: concatenate
```
Comment (Takadonet) [inline tools/concat/.shed.yml]: change description to **Concatenate paired datasets**

File: tools/concat_paired/.shed.yml [hunk 6]
```diff
@@ -0,0 +1,7 @@
+categories: [Assembly]
```
Comment (Takadonet) [inline tools/concat_paired/.shed.yml]: Switch Category to **Text Manipulation**

**Pull request merged**